google/ggrc-core
# GGRC-5185 Make ContextRBAC mixing explicit

# Dependencies

This PR is `on hold` until the dependencies are merged:

- [x] https://github.com/google/ggrc-core/pull/7823
- [x] https://github.com/google/ggrc-core/pull/7844
- [x] https://github.com/google/ggrc-core/pull/7854

# Issue description

This is a prerequisite for removing contexts from our app.

We have to make sure that every model that currently contains the context mixin still has it, but it should be explicitly listed on the model. This will enable us to start removing contexts from models one by one, which will be a lot safer than doing all models together and missing any special cases.

# Steps to test the changes

- Check that our app still works and that all models still work, Basically select a sample of all models (try to find edge cases) and check that we can create update and delete them.

- Verify that all models that had contextrbac mixin in before still have it in now. 


# Solution description

This PR moves the context rbac mixin out of our base mixin and adds
it to all models that contain the base mixin. Because in the base mixin
is included in Slugged and BusinessObject, some of the placements of the
ContextRBAC mixin are next to those and not actual Base mixin.

This PR should not change any code behavior just formatting. This
way we will be able to remove the context mixin from individual objects
one at a time.

# Sanity checklist




# PR Review checklist
